src/org/zaproxy/zap/extension/ascanrules/TestServerSideInclude.java

```diff
@@ -156,7 +166,7 @@ private boolean testServerSideInclude(String parameter, String value, Pattern te
 
             StringBuilder evidence = new StringBuilder();
             if (matchBodyPattern(message, testEvidence, evidence)) {
-                bingo(Alert.RISK_HIGH, Alert.CONFIDENCE_MEDIUM, null, parameter, evidence.toString(), value, message);
+                bingo(Alert.RISK_HIGH, Alert.CONFIDENCE_MEDIUM, null, parameter, value, value, evidence.toString(), message);

```

Comment:
Does it worth have the attack in the other info?

Reply:
I dunno, I left it as close to previous as possible. Will switch, probably no point in duplicating.

Reply:
OK.